### PR TITLE
Added pypy and pypy3 tests

### PR DESCRIPTION
**Pull request by azmeuk**
It seems that pypy and pypy3 are supported without effort :)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -11,3 +11,43 @@ cover-erase=1
...
+
+
+[testenv:docs]
+basepython =
+    python2.7
+commands =
+    sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
+deps =
+    Sphinx
+    pylons-sphinx-themes
```
Comment (tseaver) [inline setup.cfg]: Please put this content back in `tox.ini` --  moving the project into "one config file to rule them all" mode is a separate, and more controversial change than adding testing support for `pypy` / `pypy3`.

Comment (stevepiercy) [reply setup.cfg]: See how this was handled in Pyramid. https://github.com/Pylons/pyramid/commit/9c2373c3b32f587061eae387edf2678a4307897a#diff-b91f3d5bd63fcd17221b267e851608e8

Comment (azmeuk) [reply setup.cfg]: Ok I removed the commit! Sorry for that.

**Comment by tseaver** [comment]
@azmeuk Thanks for the patch!

**Pull request merged**